environment.yml

```diff
@@ -0,0 +1,20 @@
+channels:
+    - conda-forge
+    - bioconda
+    - defaults
+dependencies:
+    - bcalm=2.2.1
+    - snakemake-minimal=5.6.0
```

Comment:
@ctb needs to by `- snakemake-minimal=5.8.1`...sorry!

Reply:
fixed thx